public/video-ui/src/components/Video/Display.js

```diff
@@ -62,6 +66,37 @@ class VideoDisplay extends React.Component {
     return this.props.video.expiryDate <= Date.now();
   };
 
+    pageCreate = () => {
+
+    this.setState({
+      pageCreated: true
+    });
+
+    const metadata = {
+      title: this.props.video.title,
+      standfirst: this.props.video.description
+    };
+
+    const videoBlock = getVideoBlock(this.props.video.id, metadata);
+
+    return this.props.videoActions.createVideoPage(this.props.video.id, metadata, this.getComposerUrl(), videoBlock);
+  }
+
+  getComposerUrl = () => {
+    return getStore().getState().config.composerUrl;
+  }
+
+  renderCreateButton = () => {
+      const filterUsageType = this.props.usages.filter(value => value.type === 'video');
```

Comment:
meep, I meant for `videoPageUsages` would be more descriptive for `filterUsageType` rather than `renderCreateButton`